CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 #### Fixes
 
 * Your contribution here.
+* [#743](https://github.com/ruby-grape/grape-swagger/pull/743): CI: use 2.4.6, 2.5.5 - [@olleolleolle](https://github.com/olleolleolle).
```

Comment:
there is no entry needed for it 😉

Reply:
I agree, I listened to the robot, though.